tests/test_sanity.py

```diff
@@ -745,12 +733,6 @@ def test():
           assert 'js optimizer using native' in output
           test() # still works
 
-      finally:
```

Comment:
I was going to complain about adding a dependency but removing these try/finallys is so dang nice